examples/flower_classifier/train_many.py

```diff
@@ -0,0 +1,61 @@
+import mlflow
+
+
+epochs = "30"
+batch_size = "32"
```

Comment:
It seems like `epochs` and `batch_size` should be configurable parameters in the `MLProject` that calls this script

Reply:
Actually this a hardcoded file meant for debugging, I will remove it.